lib/evaluate.js

```diff
@@ -82,6 +86,8 @@ export const evaluate = async ({
 
 export const runFraudDetection = async (roundIndex, measurements, { fetchRoundDetails, recordTelemetry }) => {
   const roundDetails = await fetchRoundDetails(roundIndex, recordTelemetry)
+  debug(`ROUND DETAILS for round=${roundIndex}`, roundDetails)
```

Comment:
```suggestion
  debug(`ROUND DETAILS for round=%s`, roundIndex, roundDetails)
```

`debug` wants to use printf formatting, let's be consistent